textual/src/toga_textual/widgets/base.py

```diff
@@ -36,6 +36,24 @@ def __init__(self, interface):
         self.container = None
         self.create()
 
+        self._pending_children: list[Widget] = list()
```

Comment:
Is the `_pending_children` collection needed?  Is there any situation where `_pending_children` will be different to `self.interface.children`? I can't think of one... the widget is either mounted, in which case all children will be added, or the widget isn't mounted, in which case all children will *not* be mounted.

Reply:
Yeah; that's probably true. I think I was imagining the abstract case where the mounted state could change over time; this would introduce the possibility that the superset of children for the widget is not the same as the widgets that still need to be mounted on it.

Right now, though, it doesn't even work to remove a widget with children and then add that same widget back to the app; when you do this, the children of the that app are lost.

Given all this, I'll just implement the simpler design where you can mount a widget once; therefore, mounting children is only deferred once.